src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_acr_token.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+from azure_devtools.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+
+class AcrTokenCommandsTests(ScenarioTest):
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer()
+    @unittest.skip('blocked till the feature gets deployed to production')
```

Comment:
@hkuadithya, this is the test we can re-record &enable once the deployment to prod is done